### PR TITLE
LPS-12345 | master

### PR DESCRIPTION
**Pull request by Hijky**
Test for training

This is only for master

@Tim-Cao

### CODE REVIEW

**Comment by Hijky** [comment]
ci:test:sf

**Comment by Hijky** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/myfirsttest/MyFirstTest.testcase [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+		if ("${testPortalInstance}" == "true") {
+			PortalInstances.tearDownCP();
+		}
+		else {
+			User.logoutAndLoginPG(
+				userLoginEmailAddress = "test@liferay.com",
+				userLoginFullName = "Test Test");
+
+			JSONUser.tearDownNonAdminUsers();
+		}
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/myfirsttest/MyFirstTest.testcase:24]: No need

File: portal-web/test/functional/com/liferay/portalweb/tests/mysecondtest/MySecondTest.testcase [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+		if ("${testPortalInstance}" == "true") {
+			PortalInstances.tearDownCP();
+		}
+		else {
+			User.logoutAndLoginPG(
+				userLoginEmailAddress = "test@liferay.com",
+				userLoginFullName = "Test Test");
+
+			JSONUser.tearDownNonAdminUsers();
+		}
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/mysecondtest/MySecondTest.testcase:24]: No need

**Comment by Tim-Cao** [comment]
ci:close

**Pull request closed without merging**